extensions/amp-bind/amp-bind.md

```diff
@@ -76,48 +141,76 @@ The state can be initialized with the `amp-state` component:
+| `<a>` | `[href]` | Changes the link. |
+| `<button>` | `[disabled]`<br>`[type]`<br>`[value]` | Various. See [button attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Attributes). |
+| `<fieldset>` | `[disabled]` | Enables or disables the fieldset. |
+| `<input>` | `[accept]`<br>`[accessKey]`<br>`[autocomplete]`<br>`[checked]`<br>`[disabled]`<br>`[height]`<br>`[inputmode]`<br>`[max]`<br>`[maxlength]`<br>`[min]`<br>`[minlength]`<br>`[multiple]`<br>`[pattern]`<br>`[placeholder]`<br>`[readonly]`<br>`[required]`<br>`[selectiondirection]`<br>`[size]`<br>`[spellcheck]`<br>`[step]`<br>`[type]`<br>`[value]`<br>`[width]` | Various. See [input attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes). |
+| `<option>` | `[disabled]`<br>`[label]`<br>`[selected]`<br>`[value]` | Various. See [option attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#Attributes). |
+| `<optgroup>` | `[disabled]`<br>`[label]` | Various. See [optgroup attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup#Attributes). |
+| `<select>` | `[autofocus]`<br>`[disabled]`<br>`[multiple]`<br>`[required]`<br>`[size]` | Various. See [select attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#Attributes). |
+| `<source>` | `[src]`<br>`[type]` | Various. See [source attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source#Attributes). |
+| `<track>` | `[label]`<br>`[src]`<br>`[srclang]` | Various. See [track attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track#Attributes). |
+| `<textarea>` | `[autocomplete]`<br>`[autofocus]`<br>`[cols]`<br>`[disabled]`<br>`[maxlength]`<br>`[minlength]`<br>`[placeholder]`<br>`[readonly]`<br>`[required]`<br>`[rows]`<br>`[selectiondirection]`<br>`[selectionend]`<br>`[selectionstart]`<br>`[spellcheck]`<br>`[wrap]` | Various. See [textarea attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#Attributes). |
 
 ### Expressions
 
-`amp-bind` expressions are JS-like with some important differences:
+`amp-bind` expressions are similar to JavaScript with some important differences.
+
+#### Differences from JavaScript
+
+- Expressions may only access the containing document's [state](#state).
+- Expressions **do not** have access to globals like `window` or `document`.
+- Only [whitelisted functions](#whitelisted-functions) are allowed.
+- Custom functions, classes and some control flow statements (e.g. `for`) are disallowed.
+- Undefined variables and array-index-out-of-bounds return `null` instead of `undefined` or throwing errors.
+- A single expression is currently capped at 50 operands for performance reasons. Please [contact us](https://github.com/ampproject/amphtml/issues/new) if this is insufficient for your use case.
+
+#### Whitelisted functions
+
+| Object type | Function(s) | Example |
```

Comment:
Also add the whitelisted math functions:

abs, ceil, floor, max, min, random, round, sign